server/authority.go

```diff
@@ -0,0 +1,32 @@
+/*-
+ * Copyright 2016 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"net/http"
+
+	"golang.org/x/crypto/ssh"
+)
+
+func (c *context) Authority(w http.ResponseWriter, r *http.Request) {
+	if !clientAuthenticated(r) {
```

Comment:
All the other endpoints are (except _status). I think everything should just be authenticated all the time, because... why not?